### PR TITLE
cleared unwanted else clauses and minor readability improvement

### PR DESCRIPTION
**Pull request by navan0**

### CODE REVIEW

File: tensorflow/python/module/module.py [hunk 1]
```diff
@@ -349,8 +347,8 @@ def _flatten_module(module,
       if predicate(leaf):
         if with_path:
           yield module_path + leaf_path, leaf
-        else:
-          yield leaf
+
```
Comment (mihaimaruseac) [inline tensorflow/python/module/module.py]: Should remove trailing whitespace

File: tensorflow/python/module/module.py [hunk 2]
```diff
@@ -135,9 +134,8 @@ def name_scope(self):
     """Returns a `tf.name_scope` instance for this class."""
     if tf2.enabled():
       return self._name_scope
-    else:
       # In TF1 name_scope is not re-entrant in eager so we cannot memoize it.
```
Comment (mihaimaruseac) [inline tensorflow/python/module/module.py]: Should fix indentation for the comment too.

File: tensorflow/python/module/module.py (hunk 1 above)
Comment (Navan0) [reply tensorflow/python/module/module.py]: resolved

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
Please fix failures

**Comment by Navan0** [comment]
@mihaimaruseac all required builds are succefull , can you merge this?

**Comment by mihaimaruseac** [comment]
Tests are not successful

```
//tensorflow/python/module:module_test                                   FAILED in 3 out of 3 in 39.5s
  Stats over 3 runs: max = 39.5s, min = 17.3s, avg = 27.4s, dev = 9.2s
  /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/python/module/module_test/test.log
  /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/python/module/module_test/test_attempts/attempt_1.log
  /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/python/module/module_test/test_attempts/attempt_2.log
```

```
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/python/module/module_test.runfiles/org_tensorflow/tensorflow/python/module/module_test.py", line 503, in test_with_path
    mod._flatten(with_path=True, predicate=module._is_variable))
TypeError: cannot convert dictionary update sequence element #1 to a sequence

----------------------------------------------------------------------
```

**Comment by gbaned** [comment]
@Navan0  Could you please check reviewer comments and keep us posted. Thanks!

**Comment by Navan0** [comment]
@gbaned sure

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by mihaimaruseac** [comment]
@Navan0 did you fix tests? There is also a merge conflict that needs fixing.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**